### ISSUE TITLE
Cape Movement Enhancement

### ISSUE THREAD

**Issue by jurvyy** [corentingosselin/ProdigyCape#10]
**Not quite a bug**, though it could be very nice to have. Seems like the height movement of the player (jumping) is perfectly attacted to the cape. Though when a player rotates their head, the cape has a slight delay catching up. Is it possible to make it so the rotation is smoother? It would be a great addition to the plugin improving it overall.

_Below is a video that demonstrates what I am referring to:_

https://github.com/corentingosselin/ProdigyCape/assets/137111745/a3338de2-b0ca-41e0-b5f9-0732e8dde85d

**Comment by max1mde** [comment]
Not really because the movement is calculated by the client but the rotations have to be updated by the server which always has a delay.

But the movement could still be interpolated which would make it smoother.

**Comment by corentingosselin** [comment]
Yes we need to improve some part of the movements to make a almost the same movement as a real cape. I should release an update with faster execution using async task. I don't if you will notice a difference but it should be better

**Comment by max1mde** [comment]
> Yes we need to improve some part of the movements to make a almost the same movement as a real cape. I should release an update with faster execution using async task. I don't if you will notice a difference but it should be better 

I'm still not sure why you didn't set a interpolation duration on the display entity? Or is that a bug on my server?

**Comment by corentingosselin** [comment]
so we manage to fix the cape rotation attach, we still need to make it smoother and the same jump effect as real cape
